index.html

```diff
@@ -21,18 +23,23 @@
     <![endif]-->
   </head>
   <body ng-app="GiphyApp">
-    <div class="container">
-      <h1 class="text-center">Welcome to Giphy Search</h1>
+    <div class="container" ng-controller="myCtrl" infinite-scroll="next()" infinite-scroll-disabled='busy' infinite-scroll-distance="0">
+      <div class="col-xs-12" >
+        <h1 class="text-center">Welcome to Giphy Search</h1>
 
-      <form class="form-inline">
-        <div class="form-group">
-          <label for="search">Search</label>
-          <input type="text" placeholder="Search for GIFs" id="search" class="form-control">
+        <form class="form-inline" ng-submit="add()">
+          <div class="form-group">
+            <label for="search">Search</label>
+            <input type="text" placeholder="Search for GIFs" id="search" class="form-control" ng-model="search">
+          </div>
+          <button type="submit" class="btn btn-primary">Search</button>
+        </form>
+        <div id="results">
+          <div class="well" ng-repeat="result in results track by $index">
+            <img ng-src={{result.images.original.url}} />
+          </div>
         </div>
-        <button type="submit" class="btn btn-primary">Search</button>
-      </form>
-
-      <div id="results"></div>
+      </div>
     </div>
   </body>
 </html>
```

Comment:
Here is the Div element I used to host the infinite scrolling:

    <div class="row" id="results" infinite-scroll="myPagingFunction()" infinite-scroll-distance="3">
        <div class="well col-md-6" ng-repeat="giphy in giphys track by giphy.id">
            <img ng-src="{{giphy.images.fixed_height.url}}" alt="Description" />
            <a ng-href="{{giphy.images.fixed_height.url}}">link</a>
        </div>
    </div>